readthedocs/projects/tests/test_views.py

```diff
@@ -270,3 +270,25 @@ def test_download_files(self):
                 resp["X-Accel-Redirect"],
                 f"/proxito/media/{type_}/project/latest/project.{extension}",
             )
+
+
+class TestProjectAdvancedUpdate(TestCase):
+    def setUp(self):
+        self.user = get(User)
+        get(EmailAddress, email=self.user.email, user=self.user, verified=True)
+        self.project = get(Project, users=[self.user])
+
+    def test_invite_by_username(self):
```

Comment:
How did this end up here? Is this some kind of rebase/merge issue? 

edit: oh, I think the name just needs to be updated?